src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -64,6 +64,10 @@ public WorkflowMultiBranchProject(ItemGroup parent, String name) {
         return new WorkflowBranchProjectFactory();
     }
 
+    @Override protected String getSuccessfulDestination() {
+        return "computation/console";
```

Comment:
Should be using `FolderComputation.getSearchUrl` as the base, in this case resulting in `"indexing/console"`.